ui/prometheus-plugin/package.json

```diff
@@ -40,7 +40,7 @@
     "immer": "^9.0.15"
   },
   "peerDependencies": {
-    "react": "^17.0.2 || ^18.0.0"
```

Comment:
do we have a reason to remove the support of react v17 ?
I don't see any changes in this PR that required this drop, am I missing something ?

Reply:
If we want to support newer React features #1353, React 17 support will need to drop

Reply:
yeah I agree but that's not the case for the moment excepting if you are willing to do it :)

Reply:
Updated ❤️